test/e2e-tests.sh

```diff
@@ -34,29 +36,8 @@ function cluster_setup() {
   go build -v -mod=vendor ./cmd/... || return 1
 }
 
-function knative_setup() {
-  start_latest_knative_serving
-}
+export KNATIVE_SERVING_VERSION=latest
+source $(dirname $0)/e2e-common.sh
 
-# Script entry point.
-
-initialize $@
-
-header "Running tests"
-
-./kn service create hello --image gcr.io/knative-samples/helloworld-go -e TARGET=Knative || fail_test
-sleep 5
-./kn service get || fail_test
-./kn service update hello --env TARGET=kn || fail_test
-sleep 3
-./kn revision get || fail_test
-./kn service get || fail_test
-./kn service create hello --force --image gcr.io/knative-samples/helloworld-go -e TARGET=Awesome || fail_test
-./kn service create foo --force --image gcr.io/knative-samples/helloworld-go -e TARGET=foo || fail_test
-sleep 5
-./kn revision get || fail_test
-./kn service get || fail_test
-./kn service describe hello || fail_test
-./kn service delete hello || fail_test
-./kn service delete foo || fail_test
-success
+export KNATIVE_SERVING_VERSION=$PREVIOUS_RELEASE
```

Comment:
To be more concrete, my suggestion would be something along those lines:

* e2e-tests.sh:

```bash
sourcce e2e-commons.sh
for version in latest $PREVIOUS_RELEASE; do
   echon "===== Starting E2E tests for $version
   run_tests $version
done

```

* e2e-common.sh:

```bash
run_tests() {
  local $version=${1}
  export KNATIVE_SERVING_VERSION=$version
  initialize

  header "[$version]: Running tests"

  kn_tests  
}


kn_tests() {
  ./kn service create hello --image gcr.io/knative-samples/helloworld-go -e TARGET=Knative ||   fail_test
  sleep 5
  ./kn service get || fail_test
  ./kn service update hello --env TARGET=kn || fail_test
  sleep 3
  ./kn revision get || fail_test
  ./kn service get || fail_test
  ./kn service create hello --force --image gcr.io/knative-samples/helloworld-go -e   TARGET=Awesome || fail_test
  ./kn service create foo --force --image gcr.io/knative-samples/helloworld-go -e TARGET=foo ||   fail_testa
  sleep 5
  ./kn revision get || fail_test
  ./kn service get || fail_test
  ./kn service describe hello || fail_test
  ./kn service delete hello || fail_test
  ./kn service delete foo || fail_test
  success
}
```